README.md

```diff
@@ -1,45 +1,11 @@
-# Github Labeler
+**Github Issue Labeler**
```

Comment:
This should be a heading, it's not rendering properly. Please make it # and add a new line to fix this.